graph/src/components/ethereum/types.rs

```diff
@@ -177,10 +179,10 @@ pub enum EthereumBlockTriggerType {
 }
 
 impl EthereumTrigger {
-    fn transaction_index(&self) -> Option<u64> {
+    fn sort_index(&self) -> Option<u64> {
         match self {
-            // We only handle logs that are in a block and therefore have a `transaction_index`.
-            EthereumTrigger::Log(log) => Some(log.transaction_index.unwrap().as_u64()),
+            // We only handle logs that are in a block and therefore have a `log_index`.
+            EthereumTrigger::Log(log) => Some(log.log_index.unwrap().as_u64()),
```

Comment:
Doesn't this mean we will be sorting by two different types of numbers? `EthereumTrigger::Call` will be sorted by its `transaction_index` and `EthereumTrigger::Log` will be sorted by its `log_index`. That is deterministic, but it could result in odd ordering.